versions.tf

```diff
@@ -18,3 +18,8 @@ terraform {
     }
   }
 }
+
+provider "aws" {
```

Comment:
@npalm Any idea how to get rid of this provider definition which is wrong here? Without `terraform validate` fails with
```
│ Error: missing provider provider["registry.terraform.io/hashicorp/aws"].cache_bucket
│
│
```

As the module requires a provider with alias `cache_bucket`, Terraform needs a provider definition for that. May be we can inject one in the pipeline?

Reply:
Gosh, I was totally confused and created the providers in the module. Fixed that and the pipeline is green now.